lms/resources/_js_config.py

```diff
@@ -42,6 +43,24 @@ def _auth_token(self):
             self._request.lti_user
         )
 
+    @property
+    def _debug(self):
+        """
+        Return some debug information.
+
+        Currently used in the Gherkin tests.
+        """
+        debug_info = {}
+
+        if self._request.lti_user:
+            debug_info.setdefault("tags", []).append(
+                "role:instructor"
+                if self._request.lti_user.is_instructor
+                else "role:learner"
```

Comment:
Why does this use `setdefault, append`? The tags are never present before we start?

This is more of a head scratcher than `debug_info["tags"] = [...]`

Reply:
Hmm, indeed. There seems to be no reason for it. Removed